docs/development/core/server/kibana-plugin-server.responseinfo.statuscode.md

```diff
@@ -0,0 +1,11 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [kibana-plugin-server](./kibana-plugin-server.md) &gt; [ResponseInfo](./kibana-plugin-server.responseinfo.md) &gt; [statusCode](./kibana-plugin-server.responseinfo.statuscode.md)
+
+## ResponseInfo.statusCode property

```

Comment:
should I add `Pre-` prefix 🤔 ?

Reply:
Yeah I think adding the `Pre-` prefix might be good. When looking at and index of the types here, it's not obvious that this is only for pre-response interceptors.